lib/components/Modal/index.tsx

```diff
@@ -125,13 +126,16 @@ class ModalImpl extends React.PureComponent<Props, {}> {
   render() {
     const {
       classes, title, description, cancelContent, continueContent, children,
-      onCancel, onContinue, onWrapperCancel,
+      onCancel, onContinue, onWrapperCancel, onClickOutside,
       isLoading, continueDanger, disableWrapperClick, confirmButtonIsLoading,
       classNameModalContent, classNameModalBody,
     } = this.props;
 
     return (
-      <ModalWrapper onCancel={(disableWrapperClick) ? undefined : (onWrapperCancel || onCancel)}>
+      <ModalWrapper
+        onCancel={(disableWrapperClick) ? undefined : (onWrapperCancel || onCancel)}
+        onClickOutside={onClickOutside}
```

Comment:
Just noticed that we already have a `onWrapperCancel ` that probably should be used instead of creating this new attribute.

Reply:
Yeah I did try to use itat first & it seems to disable the click outside but following [this wanted](https://www.notion.so/stationhq/SPEC-054-Enable-notifications-by-default-on-all-applications-when-installed-23fe4d3a4c154902b1c1da8826275f61#26e1000e678c407faac50b41d04feb0a) it does do the trick, isn't it?

Reply:
Thanks for pointing to this @magne4000 Did changed & uniformisation to this. Please see [this answer](https://github.com/getstation/browserX/pull/927/files#r340533900).